cylc/flow/etc/syntax/cylc.lang

```diff
@@ -86,110 +86,148 @@
       <!-- To regenerate this keyword list, create a minimal flow.cylc file and run something like:
         cylc get-config my-minimal-suite | sed "/^[^=]*$/d; s/=.*//g; s/^ *//g; s/ *$//g" | sort -r | uniq | sed "s/\(.*\)/<keyword>\1<\/keyword>/g"
         -->
-        <keyword>work sub-directory</keyword>
-        <keyword>warning handler</keyword>
+                        <!--TAG_FOR_AUTO_UPDATE-->
+        <keyword>URL</keyword>
+        <keyword>UTC mode</keyword>
+        <keyword>abort if inactivity handler fails</keyword>
+        <keyword>abort if shutdown handler fails</keyword>
+        <keyword>abort if stalled handler fails</keyword>
+        <keyword>abort if startup handler fails</keyword>
+        <keyword>abort if timeout handler fails</keyword>
+        <keyword>abort on inactivity</keyword>
+        <keyword>abort on stalled</keyword>
+        <keyword>abort on timeout</keyword>
+        <keyword>aborted handler</keyword>
+        <keyword>batch submit command template</keyword>
+        <keyword>batch system</keyword>
+        <keyword>clock-expire</keyword>
+        <keyword>clock-trigger</keyword>
+        <keyword>collapsed families</keyword>
```

Comment:
I think it's meant to be sorted in reverse, see https://github.com/cylc/cylc-flow/blob/c3ecb20792276227f66d8b375bf1bffdad5143b5/cylc/flow/etc/syntax/cylc.xml#L14